src/openms/source/FORMAT/HANDLERS/MzXMLHandler.cpp

```diff
@@ -864,57 +886,42 @@ namespace OpenMS
         case InstrumentSettings::MSNSPECTRUM:
           if (spec.getInstrumentSettings().getZoomScan())
           {
-            os << "\" scanType=\"zoom";
+            os << " scanType=\"zoom\"";
           }
           else
           {
-            os << "\" scanType=\"Full";
+            os << " scanType=\"Full\"";
           }
           break;
 
         case InstrumentSettings::SIM:
-          os << "\" scanType=\"SIM";
+          os << " scanType=\"SIM\"";
           break;
 
         case InstrumentSettings::SRM:
-          os << "\" scanType=\"SRM";
+          os << " scanType=\"SRM\"";
           break;
 
         case InstrumentSettings::CRM:
-          os << "\" scanType=\"CRM";
+          os << " scanType=\"CRM\"";
           break;
 
         default:
-          os << "\" scanType=\"Full";
+          os << " scanType=\"Full\"";
           warning(STORE, String("Scan type '") + InstrumentSettings::NamesOfScanMode[spec.getInstrumentSettings().getScanMode()] + "' not supported by mzXML. Using 'Full' scan mode!");
         }
 
         // filter line
-        if (spec.metaValueExists("filter string") )
+        if (spec.metaValueExists("filter string"))
         {
-          os << "\" filterLine=\"";
-          os << writeXMLEscape ( (String)spec.getMetaValue("filter string") );
+          os << " filterLine=\"";
+          os << writeXMLEscape((String)spec.getMetaValue("filter string"));
+          os << "\"";
         }
 
-        // base peak mz (used by some programs like MAVEN), according to xsd:
```

Comment:
whats happening to the base peak m/z, can MQ not deal with it? I have added this recently and it seems that other software crashes when its not there e8a964c3dbf191d9cbbb80b3d456127bc9b3a41d